packages/taxdat/tests/testthat/test_get_loctime_combs.R

```diff
@@ -0,0 +1,66 @@
+## testing for get_loctime_combs function
+
+# single year obs, single lp
+test_that("get_loctime_combs works", { 
```

Comment:
All tests need to have different names, so I would call these something like "get_loctime_combs works for single year obs and single location-period"... etc